### DISCUSSION TITLE
It is not very clear what the enabled option does if passed to queryCache.prefetchQuery

### DISCUSSION THREAD

**Discussion by ncypme** [TanStack/query#740] [General]
If I understand correctly, the `enabled` option does nothing when passed to prefetchQuery.
e.g. the function passed to prefetchQuery will be called when passed `{ enabled:false }`.
This may be technically correct according to the documentation:

> enabled: Boolean | unknown
> Set this to false to disable this query from automatically running.

 but is not very intuitive based on its name.

Example:
https://codesandbox.io/s/xenodochial-leaf-p8jmq

  Version 2.4.15

**Comment by wolverineks** [comment]
`queryCache.prefetchQuery` does not take `enabled` as a config option
